wiki/Game_modifier/10K/id.md

```diff
@@ -0,0 +1,31 @@
+---
+stub: true
+tags:
+  - 10K
+  - mod
+  - game modifier
+---
+
+<!-- TODO:
+- add score multiplier, shortcut key, and caption values
+- Add information on when and why the 10k mod was removed -->
+
+# 10K (mod)
+
+![Ikon mod 10K](/wiki/shared/mods/10K.png "Ikon mod 10K")
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*\
+*Berbeda konteks dengan [xK (mod)](/wiki/Game_modifier/xK).*
+
+## Tentang
+
+- Jenis: Spesial
+- Mode permainan yang cocok: ![][o!m]
+
+## Deskripsi
+
+*Perhatian: Mod ini sudah tidak lagi tersedia untuk digunakan.*
+
+Mod **10K** merupakan sebuah [game modifier](/wiki/Game_modifier) untuk [osu!mania](/wiki/Game_mode/osu!mania) yang memungkinkan pemain untuk menggunakan 10 tombol pada sebuah arena bermain untuk semua [beatmap](/wiki/Beatmaps) *converted* osu!mania. Fungsinya sama persis seperti mod [xK](/wiki/Game_modifier/xK), dan dengan demikian menjadi unranked.
```

Comment:
hmm i don't think the term `converted beatmap` would work that way when translated tho, perhaps try something like this :

`...yang memungkinkan pemain untuk memainkan beatmap-beatmap osu!mania yang di-*convert* dari mode lain dengan total 10 tombol input. Fungsi mod ini sama seperti mod [xK](/wiki/Game_modifier/xK) lainnya...`